server/go.mod

```diff
@@ -1,9 +1,10 @@
-module go.dedis.ch/hbt
+module hbt-server
```

Comment:
I know you changed this as part of the Sonar code coverage, but Sonar still reports 0% coverage. I'd consider reverting to "go.dedis.ch/hbt".